spec/furniture/marketplace/carts_controller_request_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe Marketplace::CartsController, type: :request do
+  let(:marketplace) { create(:marketplace) }
+  let(:space) { marketplace.space }
+  let(:room) { marketplace.room }
+  let(:person) { create(:person) }
+
+  let(:shopper) { create(:marketplace_shopper, person: person) }
+
+  describe "#update" do
+    it "changes the delivery address" do
+      cart = create(:marketplace_cart, marketplace: marketplace, shopper: shopper)
```

Comment:
why can't the `cart` be a `let` outside the `it` block? Is it because the `cart` instance must be created fresh for every `it` block? And making `cart` a `let` would make it a shared object accessed between `it` blocks and that would make tests too coupled or possibly cause tests to be flaky if they use/modify the same `cart` object?

Reply:
Nah, it's because I didn't have multiple `it` statements; but I could pull this into a `let`/`before`/`subject` structure if ya want.

Reply:
leave it. it can be pulled out if more `it` statements are ever added.

Reply:
Wound up finding a bug during my testing, which made me want to write a second test case! So `let` it is!